sys-devel/solidity/solidity-0.4.2.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+CMAKE_MIN_VERSION="3.0.0"
```

Comment:
I got it from the CMakeLists.txt file, I was asked in the other ebuild to be as precise as possible with version requirements, so I did it here too :-)